doc/chapters/api/index.rst

```diff
@@ -18,6 +18,13 @@ object path and interface.
 * Object Path: /com/pelagicore/SoftwareContainerAgent
 * Interface: com.pelagicore.SoftwareContainerAgent
 
+Prerequisities
+--------------
+The prerequisities for each method describe the absolute minimum needed for a call to that method to
```

Comment:
The prerequisities is mostly that one has a container, but that is the minimum. It doesn't hinder using other API methods before calling the next one. Perhaps it makes little sense, in which case I should probably remove it.